nle/nethack/actions.py

```diff
@@ -20,6 +20,7 @@ class TextCharacters(enum.IntEnum):
     SPACE = ord(" ")
     APOS = ord("'")
     QUOTE = ord('"')
+    DOLLAR = ord("$")  # TODO: Add SEE* actions instead (see PR #177).
```

Comment:
This must be added below NUM_9, since this is the last action in `nethack.ACTIONS`.

Reply:
Done.